lib/adapter.test.js

```diff
@@ -0,0 +1,66 @@
+const KnexAdapter = require('./adapter.js');
+const knex = require('knex');
+const mockKnex = require('mock-knex');
+const tracker = mockKnex.getTracker();
+
+function createMockPolicy(policyValues, roleValues) {
+  // Don't alter passed values
+  policyValues = [...policyValues];
```

Comment:
I don't understand this one. You're overwriting the passed parameters here with the same value. What's the point?